packages/@aws-cdk/aws-codebuild/lib/codepipeline-artifacts.ts

```diff
@@ -0,0 +1,14 @@
+import { Artifacts } from './artifacts';
+
+/**
+ * CodePipeline Artifact definition for a CodeBuild Project.
+ * *Note*: this type cannot be used as a secondary artifact,
+ * and because of that, you're not allowed to specify an identifier for it.
+ */
+export class CodePipelineArtifacts extends Artifacts {
```

Comment:
should be on the `Artifacts` union-like, and probably internal

Reply:
This class is actually private to this package, so I don't want to expose it on `Artifacts` at all.

Reply:
In the future, we said that we will prefix file names of non-exported types with an underscore to indicate they are private.